internship2018-1/src/arrayFilters.js

```diff
@@ -0,0 +1,34 @@
+export function filterArray(numbers, min, max){
+    var counter = 0;
+    var result = [];    
+    for(var i = 0; i < numbers.length; i++){
+        if(numbers[i] > min && numbers[i] < max){            
```

Comment:
According to the task's description, **result** array can include min and max, so **>=** and **<=** operators should be used here.